### ISSUE TITLE
[Feature Request] Allow prelaunch_command to set up game folder

### ISSUE THREAD

**Issue by dliw** [lutris/lutris#2592]
I would like to use the pre-launch command  / script to set up the game folder (using overlayfs for sharing game data with a Windows installation of the game, adding mods via an overlay layer, etc.). Currently this is not possible, because lutris expects the game binary to exist before the pre-launch command is executed (file not found error). Thanks!

**Comment by strycore** [comment]
I think I understand part of this request but I'm pretty much sure that pre-launch commands is NOT the place to do this. 
Can you please re-formulate your request without taking into account the prelaunch command? This would make it easier to understand what the problem actually is and how it can be solved.

**Comment by dliw** [comment]
I will try. Essentially, my use case is as follows: I have a script that "manages" a game, meaning it sets up the game (e.g. mounting via overlayfs) and also cleans up afterwards. So
```
game_script.sh setup
wine some_mount_point/game.exe
game_script.sh teardown
```

Now I want to use Lutris to handle these game scripts, wine prefixes etc. and also get a nice launcher. So I split up the game_script.sh (#2461) (pre- and post-script) and pointed Lutris to path where `game.exe` will be *after running the setup*. However, Lutris checks the executable before running the pre-script and I get a file not found error, because `some_mount_point` does not yet exist. I hope this makes it little bit clearer.

**Comment by tannisroot** [comment]
So your core issue is essentially that your game exe doesn't have a 100% deterministic and you have to make a script to detect it and symlink to some arbitrary static path? If so, we should have an existing bug report for this.

**Comment by dliw** [comment]
Not quite. That would be the more general case. I know exactly, where the executable will be, so I can set the executable directly. It's just, that I need a script to be executed in order to create the specific mount point and don't want to do this manually.

It's not that complicated:
1. I set the executable field in lutris to `/some/mount/point/game.exe'
2. I set a pre-launch command which mounts this very specific path
3. When starting the game lutris says
``The file /some/mount/point/game.exe could not be found``

The feature request: lutris should *first* execute the pre-launch command and then check, if the executable exists. Or introduce something like a  pre-pre-lauch command.
